src/components/CategoryOverview/CategoryOverview.js

```diff
@@ -65,20 +76,30 @@ const CategoryOverview = (props) => {
             <div className={style.categoryTextWrapper}>
               <h2 className={style.categoryTitle}>{props.category.name}</h2>
 
-              <p className={style.categoryDescription}>{props.category.shortDescription}</p>
+              <p className={style.categoryDescription}>
+                {isContextCategory && props.cities
+                  ? props.category.allCitiesShortDescription
+                  : props.category.shortDescription
+                }
+              </p>
 
               <p className={style.subCategoryWrapperText}>{STRINGS.SUB_CATS_INCLUDE}</p>
 
               <div>
                 {props.category.subCategories.map((subCategory) => {
                   const url = `/${cityUrlPart}/${props.category.id}#${getSubCategorySectionId(subCategory.name)}`;
 
+                  const pillStyle = isContextCategory ? {
+                    background: COLORS[subCategory.highlightColorLight],
+                    border: `2px solid ${COLORS[subCategory.highlightColorDark]}`,
+                  } : null;
+
                   return (
                     <NavLink
                       key={subCategory.name}
                       to={url}
                     >
-                      <Pill className={style.subCategoryLink} shadow>
+                      <Pill className={style.subCategoryLink} style={pillStyle} shadow>
```

Comment:
In the designs, the shadows are the opposite way around (between the sub-categories and the link on the right).

This branch:
![image](https://user-images.githubusercontent.com/4443482/33583416-6aeb2c16-d9ae-11e7-8963-14f888add5cf.png)

Design:

![image](https://user-images.githubusercontent.com/4443482/33583424-75156062-d9ae-11e7-9a01-6fcd9c5a54db.png)